tests/test_search.py

```diff
@@ -30,7 +30,6 @@ def check_error_message(self, message):
         param('en', "Sep 03 2014"),
         param('en', "friday, 03 september 2014"),
         param('en', 'Aug 06, 2018 05:05 PM CDT'),
-
```

Comment:
I think having these blank lines made sense, is this more of a stylistic change, or required due to some automated check?

Reply:
Done 👍🏻